src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,36 @@
+package ladder.domain;
+
+import java.util.List;
+import java.util.Random;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Line {
+    private static final Random random = new Random();
+    private final List<Boolean> bridge;
+
+    public Line (int countOfPerson) {
+        this.bridge = Stream.iterate(random.nextBoolean(), this::getNextLine)
+                .limit(countOfPerson - 1)
+                .collect(Collectors.toList());
+    }
+
+    private Boolean getNextLine(Boolean previous) {
+        return !previous && random.nextBoolean();
```

Comment:
원시타입을 활용하면 어떨까요?
이펙티브 자바 아이템 61을 참고해 보시면 좋을 것 같아요 
```suggestion
    private boolean getNextLine(boolean previous) {
        return !previous && random.nextBoolean();
```